pkg/rancher-desktop/assets/specs/command-api.yaml

```diff
@@ -238,11 +238,13 @@ components:
           properties:
             adminAccess:
               type: boolean
+              x-rd-platforms: ['darwin', 'linux'] # Only in the specified platforms
```

Comment:
I'm starting to wonder if we can have a schema for the schema :p (No need to worry about it in this PR, though.)

Reply:
ok